docs/en/operation/Journal.md

```diff
@@ -70,18 +72,17 @@ backup directory can be configured by setting `alluxio.master.backup.directory`
 alluxio.master.backup.directory=/alluxio/backups
 ```
 
-See the [backup command documentation](Admin-CLI.html#backup) for additional options for specifying
-where to write a backup.
+See the [backup command documentation](Admin-CLI.html#backup) for additional backup options.
 
-## Restore
+### Restoring from a Backup
 
 To restore the Alluxio system from a journal backup, stop the system, format the
 journal, then restart the system, passing the URI of the backup with the `-i`
 (import) flag.
 
 ```bash
 $ bin/alluxio-stop.sh masters
-$ bin/alluxio formatMaster
+$ bin/alluxio formatMasters
 $ bin/alluxio-start.sh -i <backup_uri> masters
```

Comment:
I'm curious, if we do not run with passwordless ssh, and have to start each master manually, how do we import the journal?

Reply:
We would run
```
bin/alluxio-start.sh -i <backup_uri> master
```
on each master. The way `-i` works, it will only perform import if (1) the current master is primary and (2) the journal is empty. I'll update the doc to specify this